pymatgen/analysis/quasiharmonic.py

```diff
@@ -361,8 +361,7 @@ def get_summary_dict(self):
 
 @deprecated(
     replacement=QuasiHarmonicDebyeApprox,
-    message="Use PascalCased QuasiHarmonicDebyeApprox instead. "
```

Comment:
`monty.dev.deprecated` would compile the `replacement` itself:
```python
    def craft_message(
        old: Callable,
        replacement: Callable,
        message: str,
        deadline: datetime,
    ):
        msg = f"{old.__name__} is deprecated"


        if deadline is not None:
            msg += f", and will be removed on {deadline:%Y-%m-%d}\n"


        if replacement is not None:
            if isinstance(replacement, property):
                r = replacement.fget
            elif isinstance(replacement, (classmethod, staticmethod)):
                r = replacement.__func__
            else:
                r = replacement
            msg += f"; use {r.__name__} in {r.__module__} instead."


        if message:
            msg += "\n" + message
        return msg
```
So I assume there is no need to re-declare this.

Reply:
sure, feel free to change